src/controllers/pokestop.js

```diff
@@ -147,6 +145,13 @@ class Invasion extends Controller {
 				data.gruntName = data.displayTypeId && this.GameData.utilData.pokestopEvent[data.displayTypeId] ? this.GameData.utilData.pokestopEvent[data.displayTypeId] : ''
 				data.gruntType = data.displayTypeId && this.GameData.utilData.pokestopEvent[data.displayTypeId] ? this.GameData.utilData.pokestopEvent[data.displayTypeId].toLowerCase() : ''
 				data.gruntRewards = ''
+				if (data.displayTypeId === 7) {
```

Comment:
These should be in util.json - see plenty of other examples.
Maybe the color and an emoji reference should be there?

Reply:
like this?
im actually working on it 
		"3": {
			"name": "Instinct",
			"color": "FFDE00",
			"emoji": "team-instinct"
		},

Reply:
Yes, exactly